woocommerce/woocommerce-android
# Make add-ons view available for refunds

Summary
==========
Fixes issue #4772 by connecting the `RefundDetailFragment` view with the `OrderedAddonFragment`, allowing to verify the Ordered Add-on for each refunded product.

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| ![Screenshot_20210910_205700](https://user-images.githubusercontent.com/5920403/132930915-77811c90-0d2d-460e-8e34-53d708172ee5.png) | ![image](https://user-images.githubusercontent.com/5920403/132931588-6931ec62-b357-46d0-b94a-e5fd90aadbad.png) |

How to Test
==========
Prerequisites: Have your site with the add ons plugin installed, configured, and with at least one product with add-ons and at least one order of the configured product. 

1 - Launch the Woo app.
2 - Navigate to an order with at least one Product configured with Add-ons, now request a refund for this same Product.
2 - After the refund is done, in the same Order, click to see the refunded Products, verify if the `View Add-ons` Button is showing up for products that contain Add-ons inside the current Order, but doesn't show up for Products without Add-ons for that order or Products that don't have Add-ons configured at all.
3 - Click on `View Add-ons` and verify if the Ordered Addons view is displayed with all the expected data.


Update release notes:
